services/payments/interfaces.go

```diff
@@ -0,0 +1,60 @@
+package payments
+
+import (
+	"context"
+	"github.com/google/uuid"
+
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/qldb"
+	"github.com/awslabs/amazon-qldb-driver-go/v3/qldbdriver"
+)
+
+type IdempotentObject interface {
```

Comment:
do we need to publicly expose this interface?  is it needed?

Reply:
No. Unexported.